tests/engine/unit/segments/segment_operator.test.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    CONDITION_OPERATORS,
+} from '../../../../flagsmith-engine/segments/constants';
+import {
+    SegmentConditionModel,
+} from '../../../../flagsmith-engine/segments/models';
+import {traitsMatchSegmentCondition} from "../../../../flagsmith-engine/segments/evaluators";
+import {TraitModel} from "../../../../build";
```

Comment:
This import is causing your tests to fail. You need to make sure that you're not importing anything from the `build` directory since that is local to your machine only.